library.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "SBUS",
+  "keywords": "futaba, sbus, rc, remote control",
+  "description": "Futaba SBUS library for interfacing with SBUS-compatible receivers",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/Troynica/SBUS.git"
+  },
+  "authors":
+  [
+    {
+      "name": "Dennis Marinus",
+      "email": "dennis@marinus.nu",
+      "maintainer": true
+    }
+  ],
+  "dependencies":
```

Comment:
this looks like it's missing a `{`, and a matching closing one after platforms

Reply:
Yeah, aside from that, I should not have added my library.json in the first place. Removed it from the request...

Reply:
Created a new library.json after all.

Reply:
Keyword "dependencies" was missing a value. Removed in later version.